flash/base.py

```diff
@@ -639,10 +639,10 @@ def get_key(self, *args, **kwargs):
                 value = field_dict[field.attname]
             if isinstance(value, models.Model):
                 # get the pk value on instance
-                if field.rel:
-                    rel_model = field.rel.to
+                if field.remote_field:
```

Comment:
[Field.rel and Field.remote_field.to are removed](https://docs.djangoproject.com/en/2.1/releases/2.0/) on Django 2.0